examples/dyanamically-generated-background.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Trianglify - Dynamically Generated Background</title>
+
+    <style>
+      div#background {
+        width: 600px;
+        height: 120px;
+        background-size: cover;
+        background-repeat: no-repeat;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="background">
+      Let's fill this div with text adn see if that changes anything
```

Comment:
Please either remove this text or change it to something more informative

(i.e. "Using a data-uri, you can set a trianglify pattern as a background for a div that has other content")

Reply:
I'm laughing... definitely meant to take that out, can't believe I didn't catch it.